static/js/github-create-branch.js

```diff
@@ -1,11 +1,47 @@
+const createBranchPost = () => {
+	const url = "/github/create-branch";
+	// Todo improve this select2 get and split once we have real data coming in
+	const repoWithOwner = $("#ghRepo").select2('data').text.split(' / ');
+	const data = {
+		owner: repoWithOwner[0],
+		repo: repoWithOwner[1],
+		sourceBranchName: document.getElementById('ghParentBranch').value,
```

Comment:
/nit
Its a bit confusing as to why we are using `document.getElementById` in some places, but using `$("ID")` in others. Should be just using one way to maintain consistency